popgym/wrappers/last_action_wrapper.py

```diff
@@ -0,0 +1,82 @@
+from typing import Tuple, Optional
```

Comment:
This file is a bit complex. Could you maybe add a docstring with a short summary of what the wrapper does and any assumptions made?